src/modules/defi-sdk/background.ts

```diff
@@ -26,3 +26,24 @@ export function configureBackgroundClient() {
     },
   });
 }
+
+export const configureBackgroundTestClient = memoizeOne(() => {
+  invariant(DEFI_SDK_TESTNET_API_URL, 'DEFI_SDK_TESTNET_API_URL not defined');
+  invariant(DEFI_SDK_API_TOKEN, 'DEFI_SDK_API_TOKEN not defined');
+  const client = new Client({
+    url: DEFI_SDK_TESTNET_API_URL,
+    apiToken: DEFI_SDK_API_TOKEN,
```

Comment:
Would we want to use a separate token for the testnet api?

Reply:
🤷‍♀️